packages/gitbook/src/components/AIActions/AIActions.tsx

```diff
@@ -0,0 +1,188 @@
+            onClick={() => {
+                // Open the chat if it's not already open
+                if (!chat.opened) {
+                    chatController.open();
+                }
+
+                // Send the "What is this page about?" message
+                chatController.postMessage({
+                    message: tString(language, 'ai_chat_suggested_questions_about_this_page'),
+                });
+            }}
+        />
+    );
+}
+
+export function CopyMarkdown(props: { markdown: string; type: AIActionType }) {
+    const { markdown, type } = props;
+    const [copied, setCopied] = useState(false);
+
+    // Close the dropdown menu manually after the copy button is clicked
+    const closeDropdownMenu = () => {
+        const dropdownMenu = document.querySelector('div[data-radix-popper-content-wrapper]');
+
+        // Cancel if no dropdown menu is open
+        if (!dropdownMenu) return;
+
+        // Dispatch on `document` so that the event is captured by Radix's
+        // dismissable-layer listener regardless of focus location.
+        document.dispatchEvent(new KeyboardEvent('keydown', { key: 'Escape', bubbles: true }));
+    };
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={copied ? 'check' : 'copy'}
+            label={copied ? 'Copied!' : 'Copy for LLMs'}
+            description="Copy page as Markdown"
+            onClick={(e) => {
+                e.preventDefault();
+
+                if (!markdown) return;
+                navigator.clipboard.writeText(markdown);
+                setCopied(true);
+
+                setTimeout(() => {
+                    if (type === 'dropdown-menu-item') {
+                        closeDropdownMenu();
+                    }
+
+                    setCopied(false);
+                }, 2000);
+            }}
+        />
+    );
+}
+
+export function ViewAsMarkdown(props: { markdownPageUrl: string; type: AIActionType }) {
+    const { markdownPageUrl, type } = props;
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={<MarkdownIcon className="size-4 fill-current" />}
+            label="View as Markdown"
+            description="View this page as plain text"
+            href={`${markdownPageUrl}.md`}
+        />
+    );
+}
+
+export function OpenInLLM(props: {
+    provider: 'chatgpt' | 'claude';
+    url: string;
+    type: AIActionType;
+}) {
+    const { provider, url, type } = props;
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={
+                provider === 'chatgpt' ? (
+                    <ChatGPTIcon className="size-3.5 fill-current" />
+                ) : (
+                    <ClaudeIcon className="size-3.5 fill-current" />
+                )
+            }
+            label={provider === 'chatgpt' ? 'Open in ChatGPT' : 'Open in Claude'}
+            description={`Ask ${provider} about this page`}
```

Comment:
```suggestion
            description={tString(language, 'ai_chat_ask_about_page', provider)}
```
using the generic "ask about page" translation from [above](https://github.com/GitbookIO/gitbook/pull/3431/files#r2192202672).